io.catenax.sealant.sealant_pass/1.0.0/SealantPass.ttl

```diff
@@ -0,0 +1,648 @@
+
+
+:SealantPass a bamm:Aspect;
+    bamm:properties (:documents :performanceProperties :serialNumber :productIdentification :application :supplier :hazardsIdentification :productComposition :safetyMeasures :productProperties :productDisposal :transportInformation :regulatoryInformation :carbonFootprint);
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "sealant pass"@en;
+    bamm:description "The Sealant Passport represents the needs from manufacturers of chemical subtance and mixtures (e.g. BASF, Henkel). It describes data required by the safety data sheet - which is based on the REACH Regulation No 1907/2006 in the format of a DPP."@en;
+    bamm:see <https://shorturl.at/euCJ8>.
+:documents a bamm:Property;
+    bamm:characteristic :DocumentCharacteristic;
+    bamm:preferredName "documents"@en;
+    bamm:description "Set of documents containing the description of battery components."@en.
+:DocumentCharacteristic a bamm:Characteristic;
+    bamm:dataType :DocumentEntity;
+    bamm:preferredName "document characteristic"@en;
+    bamm:description "Set of documents containing the description of chemical substancen and mixture components"@en.
+:DocumentEntity a bamm:Entity;
+    bamm:properties ([
+  bamm:property :declarationOfConformity;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :removalProcedure;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :dueDiligence;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :technicalDocumentation;
+  bamm:optional "true"^^xsd:boolean
+] :fullSDS [
+  bamm:property :sustainabilityReport;
+  bamm:optional "true"^^xsd:boolean
+]);
+    bamm:preferredName "document entity"@en;
+    bamm:description "Entity encapsulating the details of a attachment for the chemical substances and mixtures of the pass."@en.
+:declarationOfConformity a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "declaration of conformity"@en;
+    bamm:description "Declaration of conformity (CE) is describing a regulatory requirement from the EU."@en.
+:removalProcedure a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "removal procedure"@en;
+    bamm:description "The documents explain the removal and possible recycling options of the product."@en.
+:dueDiligence a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "due diligence"@en;
+    bamm:description "Report on Supply Chain Due Diligence policies. Link to document or website on its supply chain due diligence policies."@en.
+:technicalDocumentation a bamm:Property;
+    bamm:characteristic :LocateCharacteristic;
+    bamm:preferredName "technical documentation"@en;
+    bamm:description "Technical documentation refers to documents that explain the use, functionality, creation, or architecture of the product."@en.
+:fullSDS a bamm:Property;
+    bamm:characteristic :SpecificsSDSCharacteristic;
```

Comment:
What is SDS. Consider expanding the acronym or at least introduce it in the description

Reply:
fixed